src/Shared/Recruit.Shared.Web/ViewModels/Skills/SkillsViewModelHelper.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
If the part and functionality within the orchestrator is really common, would it not be better if we shared those bits of the orchestrator that the orchestrator in the web projects delegate to? Might mean we don't have to have two sets of similar unit tests.

Same thing for the ViewModel and EditModel where you have created interfaces, would they not be better off having base classes. This might make the concrete classes in the web projects hollow but would still allow extensibility.

Reply:
I'll have a look and see if we can get rid of the interface.  The problem is the VMs inherit from VacancyRouteModel which is different between Provider and Employer.  I'll have another think.

I didn't want to share too much of the orchestrator code to get away from tight coupling between Provider and Employer. The building of the skills array is the bulk of the code in the orchestrator and I thought that was more shareable - E.g. When we build the API we can reuse this helper there too.